### ISSUE TITLE
Binary file data view regression

### ISSUE THREAD

**Issue by drewnoakes** [gitextensions/gitextensions#7322]
## Current behaviour

a3420a28fae2afc1ef0ae10962adcebd01613875 seems to have regressed the display of binary data in the file viewer.

## Expected behaviour

Binary hex dump still displays.

This problem was discovered while browsing code.

**Comment by gerhardol** [comment]
This is working as expected for me, the hexdump is added to summary
(this is used when viewing files only)
Changed in #6496, issue was #6434

**Comment by drewnoakes** [comment]
The pending work here then is to remove the `ToHexDump` call that runs in a task and whose output is never used.

**Comment by gerhardol** [comment]
The output is used

**Comment by gerhardol** [comment]
I have found why I first saw a problem here. If the binary format is detected at a previous stage using gitattributes, the hexdump data is not presented. The git
Will submit a fix.

Other observations:
 * Binary output data should be limited by default, like big diffs are
 * ViewTextAsync() should no longer be Async, no async methods.
 * FileHelper.IsBinaryFileName() checks gitattributes first, if text is detected then the filename extension is ignored. In the filetree, the file was just hexdumped if the attributes detected it as a textfile, but the file contents check detected it as a binary file.
